setup.py

```diff
@@ -24,7 +27,8 @@
         reqs += [ir]
 
 extras_require = {}
-for mreqs, mode in zip([optionals, mpis, nvidias], ['extras', 'mpi', 'nvidia']):
+for mreqs, mode in (zip([optionals, mpis, nvidias, testing],
+                        ['extras', 'mpi', 'nvidia', 'testd'])):
```

Comment:
Need to make `exclude` below conditional on `tests`

Reply:
Not sure I understand this. Could you issue a commit on this?